### PR TITLE
Update gem metadata

### PR DESCRIPTION
**Pull request by CDThomas**

### CODE REVIEW

File: ore-rs.gemspec [hunk 1]
```diff
@@ -16,14 +16,22 @@ Gem::Specification.new do |s|
 
   s.authors  = ["Matt Palmer"]
   s.email    = ["matt@cipherstash.com"]
-  s.homepage = "https://github.com/cipherstash-ruby-ore-rs"
+  s.homepage = "https://cipherstash.com"
```
Comment (freshtonic) [inline ore-rs.gemspec:19]: Don't we want the homepage to be the git repo?

File: ore-rs.gemspec [hunk 2]
```diff
@@ -16,14 +16,22 @@ Gem::Specification.new do |s|
...
 
   s.required_ruby_version = ">= 2.7.0"
 
+  s.metadata["homepage_uri"] = s.homepage
+  s.metadata["source_code_uri"] = "https://github.com/cipherstash/ruby-ore-rs"
+  s.metadata["changelog_uri"] = "https://github.com/cipherstash/ruby-ore-rs/releases"
+  s.metadata["bug_tracker_uri"] = "https://github.com/cipherstash/ruby-ore-rs/issues"
+  s.metadata["documentation_uri"] = "https://rubydoc.info/gems/ore-rs"
+  s.metadata["mailing_list_uri"] = "https://discuss.cipherstash.com"
+  s.metadata["funding_uri"] = "https://cipherstash.com/signup"
```
Comment (freshtonic) [inline ore-rs.gemspec]: Is this deliberate? It doesn't seem like the intended purpose of a funding URI

File: ore-rs.gemspec (hunk 1 above)
Comment (CDThomas) [reply ore-rs.gemspec:19]: From convo offline: we want this to be the ActiveStash landing page once it's ready (across all of our gems). I'm making this the CipherStash homepage until the landing page is ready.

File: ore-rs.gemspec (hunk 2 above)
Comment (CDThomas) [reply ore-rs.gemspec]: From the convo offline: I'll update this to the pricing page.

File: ore-rs.gemspec (hunk 1 above)
Comment (mpalmer) [reply ore-rs.gemspec:19]: I have very strong opinions that the homepage for gems only tangentially-related to ActiveStash should *not* be ActiveStash.  If that is the decision, though, I think it'd be best just to set that to be the homepage now, and draft this PR until that page is ready to go.  There's very little point in making a release of `ore-rs` today to change the homepage URL, and then making another one in a day or two's time to change the homepage URL again.

**Pull request merged**